pkg/primitives/vm/utils.go

```diff
@@ -148,6 +148,12 @@ func (p *Manager) prepContainer(
 
 	// can be overridden from the flist itself if exists
 	if len(imageInfo.KernelPath) != 0 {
+		// try to decompress kernel
+		err := tryDecompressKernel(imageInfo.KernelPath)
+		if err != nil {
```

Comment:
Just some cosmetics but i prefer always to do:

```go 
if err := functionThatMightFail(); err != nil {
   // handle error
}
```
Go error handling is already annoying, so this makes it a little bit compact